src/Core/src/Platform/Android/WrapperView.cs

```diff
@@ -25,8 +25,20 @@ public partial class WrapperView : PlatformWrapperView
 
 		AView _borderView;
 
+		WeakReference<IOnTouchListener> _touchListener;
```

Comment:
What keeps a strong reference to the `IOnTouchListener`? It looks like the .NET GC would collect it. Java would then try to call `TouchEventInterceptor+OnTouchListener` and won't be able to.

Does the event continue working after some GCs occur?